### PR TITLE
Events galore

### PR DESCRIPTION
**Pull request by curioussavage**
Just two more events here.

I also did default new posts to type of text since if the user doesn't type anything down there it says its a link post. This was checked on submission but I didn't like that the ui didn't reflect the state accurately before submission.

### CODE REVIEW

**Comment by curioussavage** [comment]
comment event: 

```{
    event_topic: "comment_events"
    event_ts: 1457996049975
    event_type: "cs.comment"
    payload: { domain: "localhost:4444", geoip_country: "AU", … }
    app_name: "reddit-mobile-test"
    base_url: "/r/learnprogrammingslc/comments/4afl07/this_is_a_test_muahahaha/"
    comment_body: "test 123 test 123"
    comment_fullname: "t1_d0zxo59"
    comment_id36: "0zxo59"
    domain: "localhost:4444"
    geoip_country: "AU"
    parent_fullname: "t3_4afl07"
    parent_id36: "afl07"
    post_fullname: "t3_4afl07"
    post_id36: "afl07"
    referrer_domain: "localhost:4444"
    referrer_url: "http://localhost:4444/r/learnprogrammingslc"
    sr_id36: "2m7q"
    sr_name: "learnprogrammingslc"
    user_agent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.87 Safari/537.36"
    user_id36: "gj0o7"
    user_name: "curioussavage01"
    utc_offset:
        -7
    uuid: "4fd57327-f0df-4214-aab1-f86ac6b02601"
}```

**Comment by curioussavage** [comment]
submit event:

```javascript
{
    event_topic: "submit_events"
    event_ts: 1457996222397
    event_type: "cs.submit"
    payload: { domain: "localhost:4444", geoip_country: "AU", … }
    app_name: "reddit-mobile-test"
    base_url: "/r/learnprogrammingslc/submit"
    domain: "localhost:4444"
    geoip_country: "AU"
    post_fullname: "t3_4aflzw"
    post_id36: "4aflzw"
    post_target_domain: "i.imgur.com"
    post_target_url: "http://i.imgur.com/S0qY40m.gifv"
    post_title: "This is an excellent title"
    post_type: "link"
    referrer_domain: "localhost:4444"
    referrer_url: "http://localhost:4444/r/learnprogrammingslc"
    sr_id36: "2m7q"
    sr_name: "learnprogrammingslc"
    user_agent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.87 Safari/537.36"
    user_id36: "gj0o7"
    user_name: "curioussavage01"
    utc_offset:
        -7
    uuid: "23558ade-9500-4f9c-8ed9-49a615c0106c"
}

```

**Comment by curioussavage** [comment]
:eyeglasses: @nramadas & @drunken-economist

**Comment by nramadas** [comment]
:fish: for the javascript

File: assets/js/trackingEvents.es6.js [hunk 1]
```diff
@@ -266,6 +273,50 @@ function trackingEvents(app) {
...
+      post_created_ts: props.postCreated,
+    };
+
+    return payload;
+  }
+
+  function buildSubmitData(props) {
+    const { post, user } = props;
+    const payload = {
+      ...getBasePayload(props),
```
Comment (drunken-economist) [inline assets/js/trackingEvents.es6.js]: Looks like your payload is closing out the nested JSON after the base payload. Everything but the uuid, event_ts, event_topic, and event_type should be in the payload

**Comment by drunken-economist** [comment]
Just a small fix on the payload construction 💅

File: assets/js/trackingEvents.es6.js (hunk 1 above)
Comment (curioussavage) [inline assets/js/trackingEvents.es6.js]: If you were using the example events above for reference they got copied a little wierdly from devtools. My bad just noticed. the payload is nested properly. The clientSide event lib jack wrote handles building the stuff outside the payload object.

**Comment by curioussavage** [comment]
:haircut:

**Comment by drunken-economist** [comment]
:fish:  ahhh, yeah.

**Pull request merged**